src/components/Thread/Sendbar.vue

```diff
@@ -22,14 +23,19 @@
             </div>
             <div v-mdl class="entry mdl-textfield mdl-js-textfield" :class="is_dirty">
                 <!-- eslint-disable vue/use-v-on-exact -->
-                <textarea id="message-entry" v-model="message" class="mdl-textfield__input disabled" type="text" @keydown.down.exact="onArrowUp($event)" @keydown.up.exact="onArrowDown($event)" @keydown.tab.exact="onTab($event)" @keydown.esc.exact.prevent.stop="destroyAutoComplete" @keydown.shift.enter.stop @keydown.enter.prevent.stop="dispatchSend"></textarea>
+                <textarea id="message-entry" v-model="message" class="mdl-textfield__input disabled" type="text" @click="toggleTemplates(false)" @keydown.down.exact="onArrowUp($event)" @keydown.up.exact="onArrowDown($event)" @keydown.tab.exact="onTab($event)" @keydown.esc.exact.prevent.stop="destroyAutoComplete" @keydown.shift.enter.stop @keydown.enter.prevent.stop="dispatchSend"></textarea>
                 <label class="mdl-textfield__label" for="message-entry">{{ $t('sendbar.type') }}</label>
             </div>
             <!-- fab with correct colors will be inserted here -->
             <button id="send-button" class="send mdl-button mdl-js-button mdl-button--fab mdl-button--mini-fab mdl-button--colored mdl-js-ripple-effect" :style="{ background: send_color }" @click="dispatchSend">
                 <i class="material-icons md-18 material-icons-white">send</i>
             </button>
         </div>
+        <div id="options">
```

Comment:
```suggestion
        <div id="template-wrapper">
```
This seems really vague - unless this to be used for other feature buttons I would rename this.

Reply:
named this `container` 'options' because right now there are only templates but in future we may add more things in that container.

Reply:
Can you leave a comment to that effect please?

Reply:
Pardon me but can you elaborate what you are asking ?

Reply:
Can you leave a comment inline that indicates more options will be added. Otherwise this looks like an ill named wrapper :)

Reply:
Yes sure. Thank you so much for reviewing it.